packages/golden-layout/package.json

```diff
@@ -10,6 +10,7 @@
   "source": "src/index.ts",
   "type": "module",
   "dependencies": {
+    "@deephaven/components": "file:../components",
```

Comment:
If it's really just for the theme, would be nice at some point to extract the theme from `@deephaven/components` to just `@deephaven/theme` or something.